test/e2e/network/netpol/policies.go

```diff
@@ -18,564 +18,259 @@ package netpol
 
 import (
 	"fmt"
-
 	"k8s.io/apimachinery/pkg/util/intstr"
 
 	v1 "k8s.io/api/core/v1"
 	networkingv1 "k8s.io/api/networking/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
+type SetFunc func(policy *networkingv1.NetworkPolicy)
+
+func CreateNetworkPolicy(fn ...SetFunc) *networkingv1.NetworkPolicy {
+	policy := &networkingv1.NetworkPolicy{}
+	for _, f := range fn {
+		f(policy)
+	}
+	return policy
+}
+
+func SetObjectMetaName(name string) SetFunc {
```

Comment:
The SetObjectMetaName is the same in all functions, maybe it can be moved directly to `CreateNetworkPolicy(name string, fn ...SetFunc)`

Reply:
what if I want create networkpolicy whith out object name. like this 
https://github.com/kubernetes/kubernetes/blob/7a4914014521369be9e5162b869476fdfc7ea2f9/test/e2e/network/netpol/network_policy_api.go#L53-L59

here create policy with GenerateName. I want keep it. And I can also add other direct func to offer this param.